scss/_breadcrumb.scss

```
@@ -3,6 +3,7 @@
   flex-wrap: wrap;
   padding: $breadcrumb-padding-y $breadcrumb-padding-x;
   margin-bottom: $breadcrumb-margin-bottom;
+  font-size: font-size($breadcrumb-font-size);
```

You got to use the mixin, so:
```suggestion
  @include font-size($breadcrumb-font-size);
```